.gitignore

```diff
@@ -5,14 +5,15 @@ yarn-error.log
 coverage
 .vscode
 *.log
-w3
-.w3
+wrap
```

Comment:
Noob question: What's the reason for having different generated folders being named `wrap` and `.polywrap`?  My first thought is that we should be consistent with the generated folders, but I am almost sure that I am missing something

Reply:
They are at 2 different levels of "abstraction" when thinking about Polywrap vs WRAP.

`.polywrap` is the CLI's cache directory. This is "Polywrap Toolchain Specific".

`wrap` is the wrapper's codegen folder, which contains "WRAP Standard Specific" bindings for encoding + host imports (`__wrap_subinvoke` for example).